docker/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM openresty/openresty:alpine-fat AS production-stage
```

Comment:
is this file copy from apisix-docker repo？

Reply:
Yes, copy some section from apisix-docker